src/compressionworker.ts

```diff
@@ -0,0 +1,97 @@
+import compress from "./compress";
+import { config } from "./config";
+
+export function createCompressionWorker(
+  sampleEnvelope: IEnvelope,
+  onMessage?: (e: MessageEvent) => void,
+  onError?: (e: ErrorEvent) => void
+): Worker {
+  let worker = null;
+  if (Worker) {
```

Comment:
Worker will be null and Clarity won't get activated because checkApi will return false

---
In reply to: [136482257](https://github.com/Microsoft/clarity-js/pull/28#discussion_r136482257) [](ancestors = 136482257)